<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wpsrun_dlg</class>
 <widget class="QDialog" name="wpsrun_dlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton_geogrid">
       <property name="text">
        <string>Geogrid</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Ungrib">
       <property name="text">
        <string>Ungrib</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Metgrid">
       <property name="text">
        <string>Metgrid</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Real">
       <property name="text">
        <string>Real</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_OBSPROC">
       <property name="text">
        <string>OBSPROC</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_da">
       <property name="text">
        <string>DA</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_UpdateBC">
       <property name="text">
        <string>Update BC</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_wrf">
       <property name="text">
        <string>WRF</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="textEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxMPI">
     <property name="text">
      <string>Intel MPI</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Core Number:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcdNumber">
       <property name="smallDecimalPoint">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_time_from_grib">
       <property name="text">
        <string>GetTime from GRIB</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_init">
       <property name="text">
        <string>Init</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_run">
       <property name="text">
        <string>Fire!</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
